docs/dev-tools/zombienet/index.md

```diff
@@ -0,0 +1,600 @@
+    
+- [Zombienet repository]( https://github.com/paritytech/zombienet ){target=_blanket}
+- [Element public channel]( https://matrix.to/#/!FWyuEyNvIFygLnWNMh:parity.io?via=parity.io&via=matrix.org&via=web3.foundation ){target=_blanket}
+
+## Installation
+
+
+Zombienet releases can be found on the [Zombienet repository]( https://github.com/paritytech/zombienet ){target=_blanket}. Each release includes executables for Linux and Macos, which are generated using [pkg]( https://github.com/vercel/pkg ){target=_blanket}. This allows the Zombienet CLI to operate without requiring Node.js to be installed. 
+
+In order to install Zombienet, there are some options available, depending on the user's preferences and the environment where it will be used. The following sections will guide you through the installation process for each of the available options.
+
+=== "Using the executable" 
+
+    Zombienet can be downloaded using the latest release uploaded on the [Zombienet repository]( https://github.com/paritytech/zombienet/releases ){target=_blanket}. You can download the executable for your operating system and architecture, and then move it to a directory in your PATH. Or you can either dowlonad it directly from the command line using `curl` or `wget`:
+
+    === "Curl"
+        ```bash
+        curl -LO \\
+        https://github.com/paritytech/zombienet/releases/download/<INSERT_ZOMBIENET_VERSION>/<INSERT_ZOMBIENET_EXECUTABLE>
+        ``` 
+    === "Wget"
+        ```bash
+        wget \\
+        https://github.com/paritytech/zombienet/releases/download/<INSERT_ZOMBIENET_VERSION>/<INSERT_ZOMBIENET_EXECUTABLE>
+        ```
+    !!! note
+        Ensure to replace the URL with the `<INSERT_ZOMBIENET_VERSION>` that you want to download, as well as the `<INSERT_ZOMBIENET_EXECUTABLE>` with the name of the executable file that matches your operating system and architecture. This tutorial uses v1.3.106 and zombienet-macos-arm64 as an example.
+
+    Then, you should make the downloaded file executable:
+
+    ```bash
+    chmod +x zombienet-macos-arm64
+    ```
+    Finally, you can run the following command to check if the installation was successful, if so, it will display the version of the installed Zombienet:
+
+    ```bash
+    ./zombienet-macos-arm64 version
+    ```
+
+    !!! note
+        If you want to add the `zombienet` executable to your PATH, you can move it to a directory in your PATH, such as `/usr/local/bin`:
+        ```bash
+        mv zombienet-macos-arm64 /usr/local/bin/zombienet
+        ```
+
+=== "Using Nix"
+
+    For Nix users, Zombienet repository provides a [`flake.nix`](https://github.com/paritytech/zombienet/blob/main/flake.nix){target=_blanket} file that can be used to install Zombienet using the Nix package manager. To install Zombienet using Nix, you can run the following command, that will fetch the flake and install the Zombienet package:
```

Comment:
```suggestion
    For Nix users, Zombienet repository provides a [`flake.nix`](https://github.com/paritytech/zombienet/blob/main/flake.nix){target=_blanket} file that can be used to install Zombienet. To install Zombienet using Nix, you can run the following command, that will fetch the flake and install the Zombienet package:
```

Reply:
I paraphrased this part just a bit just to make it clearer though